CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ if(MSVC)
 else()
   option(BUILD_SHARED_LIBS "Build shared libraries" ON)
 endif()
-option(KIDO_BUILD_OSGKIDO "Build osgKido library" ON)
+option(KIDO_BUILD_OSGKIDO "Build osgKido library" ON) # TODO(JS): we probably should remove this option or add another build options for other extensions as well for consistency.
```

Comment:
If I understand correctly, KIDO 0.1 is already doing exactly what you mentioned, and KIDO 0.1 and KIDO 1.0 will have same package structure.